src/Search/Comb/Comb.php

```diff
@@ -827,18 +826,11 @@ private function standardizeArray($array)
      *
      * @param  string  $query  Query to test
      *
-     * @throws NoQuery
      * @throws NotEnoughCharacters
      */
     private function testValidQuery($query)
     {
-        $length = strlen($query);
-
-        if ($length === 0) {
-            throw new NoQuery('No query given.');
-        }
-
-        if ($length < $this->min_characters) {
+        if (strlen($query) < $this->min_characters) {
```

Comment:
I think you should maybe leave it as it was, but change the `NoQuery` condition to `if ($length === 0 && $this->min_characters > 0) {`

Then things will work exactly the same for everyone, but if you set your `min_characters` config to `0`, you can go ahead and search for nothing.

Reply:
Good idea